src/main/java/org/jmhsrobotics/frc2024/utils/ControllerRumble.java

```diff
@@ -0,0 +1,42 @@
+package org.jmhsrobotics.frc2024.utils;
+
+import edu.wpi.first.wpilibj.GenericHID;
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+
+public class ControllerRumble {
+  private GenericHID controller;
+
+  public ControllerRumble(GenericHID controller) {
+    this.controller = controller;
+  }
+
+  public void rumble(int duration, double strength) {
+    for (int i = 0; i < duration; i++) {
+      controller.setRumble(GenericHID.RumbleType.kLeftRumble, strength);
+      controller.setRumble(GenericHID.RumbleType.kRightRumble, strength);
+    }
```

Comment:
This for loop does not properly set the rumble for a specified time. It will just cause the thread to hold and cause loop overruns. And the speed will very based on the processing speed of the computer it is run on. You should utilize a timer instead and check if a target time as elapsed and set the rumble state to be zero after the time as elapsed.